wagtail/search/backends/db.py

```diff
@@ -1,214 +1,4 @@
-from collections import OrderedDict
```

Comment:
You can replace this file with:

```python
import sys
from wagtail.utils.deprecation import MovedDefinitionHandler, RemovedInWagtail216Warning

MOVED_DEFINITIONS = {
    'DatabaseSearchQueryCompiler': ('wagtail.search.backends.database', 'DatabaseSearchQueryCompiler'),
    'DatabaseSearchResults': ('wagtail.search.backends.database', 'DatabaseSearchResults'),
    'DatabaseSearchBackend': ('wagtail.search.backends.database', 'DatabaseSearchBackend'),
    'SearchBackend': ('wagtail.search.backends.database', 'SearchBackend'),
}

sys.modules[__name__] = MovedDefinitionHandler(sys.modules[__name__], MOVED_DEFINITIONS, RemovedInWagtail216Warning)
```
Which will re-export those classes here, and raise a deprecation warning if they are imported

Reply:
I've changed that; however, I'm not able to see the deprecation warning... Is that normal? Maybe I'm not getting the deprecation warning because we're not in Wagtail 2.16 yet?

Reply:
And are we sure we want to actually deprecate the `wagtail/search/backends/db` for the `wagtail/search/backends/database` imports?